### DISCUSSION TITLE
Installation issues with Python 3.7 on Google Colab

### DISCUSSION THREAD

**Discussion by AdarshMJ** [pyg-team/pytorch_geometric#5901] [Q&A] [answered]
Hi @rusty1s @EdisonLeeeee 

I am trying to install PyG on Colab via -

```python
import os
import torch
os.environ['TORCH'] = torch.__version__
print(torch.__version__)
!pip install -q torch-scatter -f https://data.pyg.org/whl/torch-${TORCH}.html
!pip install -q torch-sparse -f https://data.pyg.org/whl/torch-${TORCH}.html
!pip install -q git+https://github.com/pyg-team/pytorch_geometric.git
```

I get -

```
1.12.1+cu113
     |████████████████████████████████| 7.9 MB 4.0 MB/s 
     |████████████████████████████████| 3.5 MB 4.1 MB/s 
  Building wheel for torch-geometric (setup.py) ... done
```

Now when I try to use -

```Python
import torch
from torch_geometric.datasets import TUDataset, Planetoid,WebKB,Actor
from torch_geometric.transforms import NormalizeFeatures
dataset = Planetoid(root = 'data/Cora',name='Cora',transform=NormalizeFeatures())
```
I get an error -
![Screenshot 2022-11-04 at 16 43 33](https://user-images.githubusercontent.com/24205200/200017287-fd79d1d6-c108-4173-bc1c-7b925242a7bb.png)

Im supposing this is because Colab still runs on Python 3.7? I tried upgrading the python version to 3.9 but it requires me to manually install all the packages, even pip. Any suggestions to solve this?

**Comment by EdisonLeeeee** [comment]
Hi @AdarshMJ 

This bug seems related to https://github.com/pyg-team/pytorch_geometric/pull/5537#issuecomment-1303381210. For now, I would suggest you to install PyG via `pip install torch_geometric` rather than from source.

**Reply by AdarshMJ** [reply]
Okay thank you. I will try that

**Comment by rusty1s** [comment, marked as answer]
Fixed via https://github.com/pyg-team/pytorch_geometric/pull/5902.

**Reply by AdarshMJ** [reply]
Thank you!
